resources/resource_ids.spec

```diff
@@ -221,5 +221,9 @@
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-brave_player/brave_player.grd": {
     "META": {"sizes": {"includes": [10]}},
     "includes": [61010],
-  }
+  },
+  "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-ai_rewriter_ui/ai_rewriter_ui.grd": {
+    "META": {"sizes": {"includes": [200]}},
```

Comment:
do we really need 200 items here? we had an issue where we ran out of the ids, because we wrote hundreds everywhere.
same question goes to `ai_chat_ui.grd`.

you can lower the value and rebuild. If it won't be enough, the build should fail.

see https://github.com/brave/brave-core/commit/7c77d1a3b1e66fc6ca798daee2364ca75d4123f1

Reply:
Good call! I just copied the AI Chat one.